docs/en-US/component/notification.md

```diff
@@ -111,7 +111,7 @@ ElNotification({}, appContext)
 | Name                     | Description                                                                                                        | Type                                                                  | Default   |
 | ------------------------ | ------------------------------------------------------------------------------------------------------------------ | --------------------------------------------------------------------- | --------- |
 | title                    | title                                                                                                              | ^[string]                                                             | ''        |
-| message                  | description text                                                                                                   | ^[string] / ^[VNode]                                                  | ''        |
+| message                  | description text                                                                                                   | ^[string] / ^[VNode] / ^[Function]`() => VNode`                       | ''        |
```

Comment:
Perhaps we can add an example to the document, and then add a version tag after the example to explain the version that supports configuring the parameter value as a function.

Reply:
![image](https://github.com/user-attachments/assets/1bf50e58-9f7d-4ba2-8b98-ad966fbf4427)

I added this example, is that okay?